src/configure/test/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as glob from 'glob';
+import * as Mocha from 'mocha';
+import * as path from 'path';
+
+export function run(): Promise<void> {
+	// Create the mocha test
+	const mocha = new Mocha({
+		ui: 'bdd',
```

Comment:
what does `bdd` mean here?

Reply:
It is the name of the interface. BDD = Behaviour Driven Development.